bin/submitTransaction.js

```diff
@@ -168,4 +169,40 @@ const submitTransaction = (net = 'preview') => {
     });
 }
 
-export { queryUTXO, createDraftTransaction, calculateTransactionFee, buildRealTransaction, signdRealTransaction, submitTransaction }
\ No newline at end of file
+const getSignedTxTransactionId = () => {
+    return new Promise((resolve, reject) => {
+        exec(`cardano-cli transaction txid --tx-file tx.signed`, (error, stdout, stderr) => {
+            if (error) {
+                reject(error.message)
+                return;
+            }
+            if (stderr) {
+                reject(stderr)
+                return;
+            }
+            resolve(stdout)
+        });
+    });
+}
+
+
+const cleanupTransactionFiles = () => {
+    return new Promise((resolve, reject) => {
+        exec(`rm tx.draft tx.signed`, (error, stdout, stderr) => {
+            if (error) {
+                reject(error.message)
+                return;
+            }
+            if (stderr) {
+                reject(stderr)
+                return;
+            }
+            resolve(true)
+        });
+    });
+}
```

Comment:
Please add a [rimraf](https://www.npmjs.com/package/rimraf) and use it instead of rm as it provides cross-platform support.
```suggestion
const cleanupTransactionFiles = () => rimraf(['tx.draft', 'tx.signed']);
```

Reply:
Fixed on d41e940